src/modules/ui/conditions.scss

```diff
@@ -0,0 +1,71 @@
+/**
+ * The challenge here to support "playable queries" and "direction" at the same time and allow mixins like:
+ *   @include apply(max-width-550())
+ *   @include apply(max-width-550(), ltr())
+ *   @include apply(max-width-550(), rtl())
+ */
+
+@function max-width($value) {
+  @return '[data-playable-max-width~="#{$value}"]'
+}
+
+@function direction-query($direction) {
+  @return if($direction, '[data-playable-dir="#{$direction}"]', '')
```

Comment:
not sure if this condition is required here. If we use `direction-query`, its strange to not pass `$direction`

Reply:
agree, removed